product-service/src/libs/lambda.ts

```diff
@@ -0,0 +1,6 @@
+import middy from "@middy/core"
+import middyJsonBodyParser from "@middy/http-json-body-parser"
+
```

Comment:
need to fix cors error
![Screenshot from 2022-07-19 17-14-45](https://user-images.githubusercontent.com/380645/179772669-ae00fe6d-eca2-4c19-adb7-b4cc2a57f6f9.png)
how to:
```js
import middy from '@middy/core'
import bodyParser from '@middy/http-json-body-parser'
import cors from '@middy/http-cors'

export const middyfy = (handler: any) => {
  return middy(handler).use(cors()).use(bodyParser())
}
```